test/test_mount.rs

```diff
@@ -203,6 +204,31 @@ exit 23";
         assert_eq!(buf, SCRIPT_CONTENTS);
     }
 
+    pub(crate) fn test_open_tree() {
```

Comment:
Nit: would like to change this to `pub fn test_open_tree()` so that it could be consistent with the other functions in this module :)